src/App.tsx

```diff
@@ -1,11 +1,40 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { Fragment } from 'react';
+import { StyleSheet, SafeAreaView, View, AppRegistry } from 'react-native';
+import Amplify from 'aws-amplify';
+import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client';
+import awsconfig from './aws-exports';
+
+import AuthContainer from './components/auth-container';
+import SplashScreen from './components/splash-screen';
+
+/*
+   Analytics manually disabled so no event is fired
+   on sign-in/sign-out. Else, AWS analytics tries to
+   publish an event with no analytics config.
+*/
+Amplify.configure({
+  ...awsconfig,
+  Analytics: {
+    disabled: true,
+  },
+});
+
+// Initialize Apollo Client
+const client = new ApolloClient({
+  uri: 'localhost:4000/graphql',
```

Comment:
Can we load the path from an environment variable instead of hardcoding it here?

Reply:
> Overall everything looks great!
> 
> Just a couple of small things to do:
> 
> * Delete `amplify/README.md` file
> * Add `src/.eslintcache` to `.gitignore`
> * Load graphql uri prefix from an environment variables
> 
> Some questions:
> 
> * Can you explain what the `testFunction` lambda function does for the sign-up and login flow? (Seems like it is not doing anything based on the implementation)
> * From the demos, are all the passwords empty?

We'll get on the changes! As for the first question, it seems to be an auto-generated function that none of us caught. With regard to the demos, the passwords are not empty. It seems the ios screen record ignores text input for these fields as a built-in security measure. I tested this myself and can confirm that text is indeed entered but the screen capture doesn't record it.